workspaces/ui-v2/src/components/ShapeRenderer/ShapeRenderer.tsx

```diff
@@ -14,7 +14,11 @@ export const ShapeRenderer: FC<ShapeRendererProps> = ({
 }) => {
   return (
     <ShapeRenderStore showExamples={showExamples}>
-      <RenderRootShape shape={shape} />
+      {shape.length > 1 ? (
+        <OneOfRender shapes={shape} parentShapeId={'root'} />
```

Comment:
This isn't necessarily true, but the usage of this component means that we don't _actually_ care what the real parentShapeId is, it's the root of the rendered component (which is used in shapeRendererStore).